CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ _released `2025-01-14`_
 ### ✨ Added
 
 - Introduce EMU_MPS device type
+- Move documentation to a web-based site
```

Comment:
in theory this should be moved to a "changed" sub-section

Reply:
Done, and I put it in a "Not released" section.